src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -30,18 +45,20 @@ public ClientInfo() {
         mOfflineBufferAvailabilityTimeout = 0L;
         mLogLevel = DEFAULT_LOG_LEVEL;
         mLogMetric = true;
+        mAutomaticStreamingFlags = automaticStreamingFlags.AUTOMATIC_STREAMING_INTERMITTENT_PRODUCER;
     }
 
     public ClientInfo(final long createClientTimeout, final long createStreamTimeout, final long stopStreamTimeout,
```

Comment:
Let us create another parameterized constructor that would take in `automaticStreamingFlags` along with the existing ones. If customers are already using the constructor , adding `automaticStreamingFlags` to the existing one would break their application.


So, in a nutshell I am suggesting 3 constructors:

1. `ClientInfo()`
2. `ClientInfo(final long createClientTimeout, final long createStreamTimeout, final long stopStreamTimeout, final long offlineBufferAvailabilityTimeout, final int logLevel, final boolean logMetric)`
3. `ClientInfo(final long createClientTimeout, final long createStreamTimeout, final long stopStreamTimeout, final long offlineBufferAvailabilityTimeout, final int logLevel, final boolean logMetric, final automaticStreamingFlags flag)`

Reply:
Added another constructor in the latest commit. We have 3 constructors as suggested now. Thanks for pointing it out!